Command/PopulateCommand.php

```diff
@@ -265,7 +266,7 @@ function(PostAsyncInsertObjectsEvent $event) use ($loggerClosure) {
         $options['indexName'] = $index;
         $options['typeName'] = $type;
 
-        $this->pagerPersisterRegistry->insert($pager, $options);
```

Comment:
this should be `$this->pagerPersister->insert($pager, $options);`

Reply:
pagerPersister is set in initialize method

Reply:
🆒